python/lsst/ts/ATHexapod/gcserror.py

```diff
@@ -32,453 +32,229 @@
 class PIError(enum.Enum):
     """Provide enum for PI Errors."""
     E_1_COM_ERROR = -1
```

Comment:
Why are you removing the documentation strings for this file?

Reply:
Created ticket.